R/utils.R

```diff
@@ -1,5 +1,21 @@
+#' HTTP Date String
+#'
+#' Given a POSIXct object, return a date string in the format required for a
+#' HTTP Date header. For example: "Wed, 21 Oct 2015 07:28:00 GMT"
+#'
+#' @noRd
+http_date_string <- function(time) {
+  lt <- as.POSIXlt(time, tz = "UTC")
```

Comment:
Being consistent with timezone at end of function

```suggestion
  lt <- as.POSIXlt(time, tz = "GMT")
```